ethbloom/src/lib.rs

```diff
@@ -224,6 +233,19 @@ impl<'a> From<&'a Bloom> for BloomRef<'a> {
 	}
 }
 
+impl Serialize for Bloom {
```

Comment:
This should be conditional, no?

Perhaps better do:
```
#[cfg(feature="serde")]
mod serialization {
  super::*;
  extern crate serde;

  impl serde::Serialize for Bloom {...]
}

Reply:
yes, added